main.scss

```diff
@@ -1,2 +0,0 @@
-@import "./src/email-only-signup";

```

Comment:
@onishiweb I have updated this now and also moved the dependency imports into `main.scss`